cmd/ethrex/cli.rs

```diff
@@ -241,6 +242,8 @@ pub enum Subcommand {
         path: String,
         #[arg(long = "removedb", action = ArgAction::SetTrue)]
         removedb: bool,
+        #[arg(long = "num-blocks")]
+        num_blocks: Option<NonZero<usize>>,
```

Comment:
instead of num block, what about start-block and end-block or something like that?

Reply:
BTW, the correct way would be to have to files, one with 0-2500 and a second one from 2500-5000. Here you have a guide as to how to achieve it: https://www.notion.so/lambdaclass/Importing-blocks-from-public-networks-200b94624713808a8456ef5810f1a88a

Reply:
I defer that to you, this sounds like a hack tbh but Im not strongly against it.